pkg/operatorsource/configuring_test.go

```diff
@@ -39,15 +39,15 @@ func TestReconcile_NotConfigured_NewCatalogConfigSourceObjectCreated(t *testing.
 	namespacedName := types.NamespacedName{Name: "foo", Namespace: "marketplace"}
 
 	// We expect that the given CatalogConfigSource object does not exist.
-	cscGet := helperNewCatalogSourceConfig(opsrcIn.Namespace, opsrcIn.Name)
+	cscGet := &v1alpha1.CatalogSourceConfig{}
```

Comment:
This makes sense to me; so a call to `client.Get` populates the provided reference, correct? So populating it with the helper isn't necessary since it will be overwritten.

Reply:
This is the sort of explanation that should go into the commit message.